pkgs/applications/misc/bb/default.nix

```diff
@@ -9,19 +9,22 @@ stdenv.mkDerivation rec {
     sha256 = "1i411glxh7g4pfg4gw826lpwngi89yrbmxac8jmnsfvrfb48hgbr";
   };
 
+  nativeBuildInputs = [ autoreconfHook ];
+
   buildInputs = [
     aalib ncurses libmikmod
     xorg.libXau xorg.libXdmcp xorg.libX11
-  ] ++ lib.optional stdenv.hostPlatform.isDarwin darwin.apple_sdk.frameworks.CoreAudio;
+  ];
 
-  postPatch = lib.optionalString stdenv.hostPlatform.isDarwin ''
-    sed -i -e '/^#include <malloc.h>$/d' *.c
+  postPatch = ''
```

Comment:
This sed tweaking is starting to become a little too complex, wouldn't it be better just write a patch?

Reply:
not really -- the patch is much harder to read while the replaces are pretty clear. get rid of malloc for stdlib and then add a couple of include files.  I updated the PR to use the patch but I'm going to revert that change cause it adds a 145 line diff.

Reply:
reverted update.

Reply:
tho I didn't realize you are the package maintainer -- if you'd prefer the patch I can re-push that change.

Reply:
Uhm, It's not that big, but ok.

Reply:
ok -- put the patch back.